<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExchangeModeWidget</class>
 <widget class="QWidget" name="ExchangeModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="orderModeBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>挂单模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>挂单中心</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>115</y>
     <width>650</width>
     <height>100</height>
    </rect>
   </property>
   <property name="WidgetStyle" stdset="0">
    <string>container2</string>
   </property>
   <widget class="QLabel" name="currentPairLabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 11px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>交易对</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentPairLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 11px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentPairLabel2_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>可用</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="availableLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(83,61,138);font:bold 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>123</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="availableLabel2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(83,61,138);font:bold 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>123</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentPairLabel2_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>可用</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="ableToBuyLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(83,61,138);font:bold 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>123</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentPairLabel2_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>可买</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="ableToSellLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(83,61,138);font:bold 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>123</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentPairLabel2_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 14px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>可卖</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="marketBtn1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>HX交易区</string>
   </property>
   <property name="BtnStyle" stdset="0">
    <string>dropDown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="marketBtn2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>BTC交易区</string>
   </property>
   <property name="BtnStyle" stdset="0">
    <string>dropDown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="marketBtn3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PAX交易区</string>
   </property>
   <property name="BtnStyle" stdset="0">
    <string>dropDown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="favoriteMarketBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>100</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>常用交易区</string>
   </property>
   <property name="BtnStyle" stdset="0">
    <string>dropDown</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>225</y>
     <width>440</width>
     <height>248</height>
    </rect>
   </property>
   <property name="WidgetStyle" stdset="0">
    <string>container2</string>
   </property>
   <widget class="QLineEdit" name="buyPriceLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="LineEditStyle" stdset="0">
     <string>red</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sellPriceLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>15</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="LineEditStyle" stdset="0">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="buyAmountLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="LineEditStyle" stdset="0">
     <string>red</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sellAmountLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>65</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="LineEditStyle" stdset="0">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLabel" name="buyPriceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 11px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>买入价</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="buyAmountLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>69</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 11px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>买入量</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="sellPriceLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>19</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 11px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>卖出价</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="sellAmountLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>69</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: rgb(137,129,161);font: 11px &quot;微软雅黑&quot;;}
</string>
    </property>
    <property name="text">
     <string>卖出量</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QToolButton" name="buyBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 0, 94);border:none;border-radius:6px;color: white;
font: 12px &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>我要买入</string>
    </property>
   </widget>
   <widget class="QToolButton" name="sellBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44,202,148);border:none;border-radius:6px;color: white;
font: 12px &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>我要卖出</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>225</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QComboBox" name="depthComboBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>225</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
